doc/source/getting_started/Installation.rst

```diff
@@ -20,11 +20,11 @@ Using pip
 
     pip install arviz
 
-Use the below pip command to install ArviZ with all of it's :ref:`Optional-dependencies`.
+Use the below pip command to install ArviZ with all of its :ref:`Optional-dependencies`.
 
 .. code:: bash
 
-    pip install arviz[all]
+    pip install arviz all
```

Comment:
This doesn't work for me at all. I think this change is motivated by zsh shells not working with `pip install arviz[all]` unlike bash shells that do work. I think the fix for that is `pip install "arviz[all]"` instead

Reply:
Done.

Reply:
just rechecked that and `pip install arviz[all]` doesn't work on a zsh shell. It has to be `pip install "arviz[all]"`

Reply:
Yes, I tried it too. Changed, it is working. Thank you.